juju/juju
# [JUJU-2233] remove charmstore pkg

Remove charmstore package from juju. Contained client and macaroon code. 

2 tiny pieces of macaroons were moved to a new package in core to be used by the controller and CMR via the bakery. The CMR api has a new MacacroonCache so the one in charmstore was safe to remove.

Part of removing charm store support from juju. 

## QA steps

There should be no impact to current behavior. However ensure that CMR still works due to macaroon var changes.

```sh
(cd tests  ; ./main.sh  resources)
(cd tests  ; ./main.sh  deploy test_deploy_bundles)
```
